web_poet/pages.py

```diff
@@ -41,16 +41,13 @@ def is_injectable(cls: typing.Any) -> bool:
 
 
 class Returns(typing.Generic[ItemT]):
-    """Inherit from this generic mixin to change the item type used by
+    """Inherit from this generic mixin to change the Item Class used by
```

Comment:
A nitpick: I'm not sure about the capitalization change here. In Scrapy and itemadapter docs "Item" usually means scrapy.Item, and "item" means an item, something which is supported by itemadapter.

Reply:
I've made these terms in Proper Case to indicate the that we're talking about the concept of "Item Classes" in the perspective of `@handle_urls` and Page Objects.

Using "item class" could simply mean that its a literal class and could lack the concept of rules, the class that's returned, etc.

Reply:
Hey! Going back to it :) In itemadapter and in Scrapy a lower-case "item class" is used to talk about a concept; Item class means `scrapy.Item` class, or maybe some other concrete item class. Upper-case Item can't be used in itemadapter, because it would cause confusion with scrapy.Item class. So, it's the opposite approach in itemadapter docs.

Reply:
> Upper-case Item can't be used in itemadapter, because it would cause confusion with scrapy.Item class. So, it's the opposite approach in itemadapter docs.

Gotcha. I just re-read https://github.com/scrapy/itemadapter 👍 . Updated in https://github.com/scrapinghub/web-poet/pull/84/commits/383b4f74261388d149138804f4d0d41791f22691